devices/tplink_re200-v4.yml

```diff
@@ -0,0 +1,23 @@
+vendor: TP-Link
+model: RE200
+variant: v4
+bootloader: unknown
+cpu: MediaTek MT7628AN
+cpu_cores: 1
+cpu_mhz: 580
+memory_mb: 64
+storage:
+  - type: unknown
+    size_mb: 8
+device_type: WiFi Router
+ethernet_100m_ports: 1
+jtag: false
+serial: 57600 / 8N1
+outdoor: false
+supported_current_rel: none
+supported_since_rel: snapshot
+target: ramips
+vlan: true
+wlan_24ghz: b/g
+wlan_driver: unknown
+wlan_hardware: unknown"
```

Comment:
I wonder whether this can/should be grouped. After all, we will have modes, driver and hardware for each band. A valuable addition would be number and type of antennas btw (also per radio).

Reply:
I agree, just like `storage` this should be grouped

Reply:
Updated, now it's an object with freq_ghz, driver and hardware.